src/main/java/uk/gov/pay/ledger/event/model/Event.java

```diff
@@ -1,19 +1,64 @@
 package uk.gov.pay.ledger.event.model;
 
-import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.databind.PropertyNamingStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import uk.gov.pay.commons.api.json.ApiResponseDateTimeSerializer;
 
+import java.time.ZonedDateTime;
+
+@JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy.class)
 public class Event {
-    private String id;
 
-    public Event() {
-    }
+    @JsonIgnore
+    private Long id;
+    private String sqsMessageId;
+    private ResourceType resourceType;
+    private String resourceExternalId;
+    @JsonSerialize(using = ApiResponseDateTimeSerializer.class)
```

Comment:
not a biggie, but we're storing [microsecond precision](https://github.com/alphagov/pay-connector/blob/master/src/main/java/uk/gov/pay/connector/events/MicrosecondPrecisionDateTimeSerializer.java) so probably we should emit that